governance/votes/Executive vote - July 2, 2021.md

```diff
@@ -0,0 +1,144 @@
+
+**Core Unit Budget Distributions - July**  
+DAI will be distributed to the Core Units as specified within their ratified budget proposals.
+
+| Core Unit | Distribution |
+|-|-|
+| Growth (GRO-001) | 126,117 DAI|
+| Content Production (MKT-001) | 44,375 DAI|
+| GovAlpha (GOV-001) | 273,334 DAI|
+| Real World Finance (RWF-001) | 155,000 DAI|
+| Risk (RISK-001) | 182,000 |
+| Protocol Engineering (PE-001) | 510,000 DAI|
+| Oracles (ORA-001) | 419,677 DAI|
+| Oracles (ORA-001) - Continuity Reserve | 800,000 DAI|
+| **Total** | **2,510,503 DAI** |
+
+**[Stability Fee](https://community-development.makerdao.com/en/learn/governance/param-stability-fee) Decreases**
+
+| Vault Type | Current Rate | Proposed Rate | Change |
+|-|-|-|-|
+| ETH-A | 3.50% | 2.00% | -1.50% |
+| ETH-B | 9.00% | 6.00% | -3.00% |
+| ETH-C | 1.00% | 0.50% | -0.50% |
+| WBTC-A | 3.50% | 2.00% | -1.50% |
+| LINK-A | 4.00% | 1.00% | -3.00% |
+| YFI-A | 4.00% | 1.00% | -3.00% |
+| UNI-A | 2.00% | 1.00% | -1.00% |
+| AAVE-A | 2.00% | 1.00% | -1.00% |
+| RENBTC-A | 4.00% | 2.00% | -2.00% |
+| COMP-A | 3.00% | 1.00% | -2.00% |
+| BAL-A | 2.00% | 1.00% | -1.00% |
+| UNIV2DAIETH-A | 2.50% | 1.50% | -1.00% |
+| UNIV2USDCETH-A | 3.50% | 2.00% | -1.50% |
+| UNIV2DAIUSDC-A | 0.50% | 0.00% | -0.50% |
+| UNIV2WBTCETH-A | 3.50% | 2.00% | -1.50% |
+| UNIV2UNIETH-A | 4.00% | 2.00% | -2.00% |
+| UNIV2ETHUSDT-A | 4.00% | 2.00% | -2.00% |
+
+**Other Changes**
+
+Activate the [Debt Ceiling Instant Access Module](https://community-development.makerdao.com/en/learn/governance/module-dciam) for the PSM-USDC-A vault type with the following parameters:
+- Maximum Debt Ceiling (`line`) = 10 billion DAI.
+- Target Available Debt (`gap`) = 1 billion DAI.
+- Ceiling Increase Cooldown (`ttl`) = 24 hours.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal **does not** include an office-hours modifier and can be executed immediately after the GSM Pause Delay has expired.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### July Core Unit Budget Distributions
+
+Due to the positive outcomes of multiple [Core Unit Budget Proposals](https://mips.makerdao.com/mips/list?&search=$AND(%23core-unit,%20%23budget,%20%23active)&subproposalsMode=true), the following budget allocations will be made to fund operations in the month of July if this executive proposal passes.
```

Comment:
Link comes up with no MIPs in the portal

Reply:
Ahhh, it got removed. I guess I'll just get rid of the link.